ipset.go

```diff
@@ -99,6 +112,23 @@ func (s *IPSet) RemoveSet(b *IPSet) {
 	}
 }
 
+// Invert updates s to contain the opposite of is current contents.
+func (s *IPSet) Invert() {
+	s.toInOnly()
+	s.out = s.in
+	s.in = []IPRange{
+		IPPrefix{IP: IPv4(0,0,0,0), Bits: 0}.Range(),
```

Comment:
Is this gofmt'd? I'd expect spaces between the IPv4 parameters.

Reply:
Oops, fixed. My NixOS was missing goimports in the env.